README.md

```diff
@@ -111,6 +114,7 @@ Organe der Hochschulgruppe sind der Vorstand und die Mitgliederversammlung.
 
 1. Änderungen dieser Satzung sind nur durch die Mitgliederversammlung möglich.
 2. Beschlüsse über eine Änderung der Satzung benötigen die Mehrheit von drei Vierteln der anwesenden Mitglieder.
+3. Die Namen der Vorstandsmitglieder in der Satzung können jederzeit ohne Mitgliederversammlung auf den aktuellen Vorstand geändert werden.
```

Comment:
```suggestion
```
This is not valid due to [BGB § 33](https://www.gesetze-im-internet.de/bgb/__33.html).